src/stores/AccountController.ts

```diff
@@ -450,3 +449,19 @@ function useLoginContext<T>(callback: (args: { cleengId: string; cleengSandbox:
 
   return useConfig((config) => callback({ ...config, customerId: user.id, auth }));
 }
+
+function withAccountService<T>(
```

Comment:
fyi, I'm not set on this pattern if you find a cleaner, easier path to take. It was just my first attempt at refactoring as quick as I could yesterday.  It's also fine for us to leave it this way and circle back to refactor to something better later.

Reply:
It's okay for now, maybe we can think of refactor once we dive more deeply into the project.